utils/bots/CoreBot/cogs/Blacklist.py

```diff
@@ -71,11 +71,10 @@ async def list(self, interaction: discord.Interaction):
         await interaction.response.defer(thinking=True)
         empty_list = []
         for p in database.Blacklist:
-            try:
-                user = self.bot.get_user(p.id)
-                empty_list.append(f"`{user.name}` -> `{user.id}`")
-            except:
-                empty_list.append(f"`{p}`")
```

Comment:
Doing standalone `p` will give the  full object, maybe add p.discordID instead?

Reply:
![image](https://user-images.githubusercontent.com/88685021/162093819-81574f64-3751-4ae3-bf4c-53f4bcadf799.png)
on my end it looks like this?